.github/workflows/python-package.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
```

Comment:
Do we not want to keep .6, .7 for backwards compatibility?

Reply:
Python versions 3.6 and 3.7 have reached their end-of-life. And i doubt aiohttp 3.9.0 or newer supports Python 3.7.

Reply:
This is confirmed with https://github.com/aio-libs/aiohttp/pull/7336

Reply:
Ok!